Shopify/shopify-cli
# Support 3.1

### WHY are these changes introduced?

Up to now, we haven't offered Ruby 3.1 support.



### WHAT is this pull request doing?



Updates 1 gem which needed an update. Otherwise everything stays the same.

I also added 3.1.0 to CI, and - subject to approval - I suggest we move the local dev environment for the CLI to 3.1.0.

### How to test your changes?



Pull the branch, `dev up` or `gem install`, run tests!

We also should look at some basic flows and make sure nothing surprises us by being broken.

### Post-release steps



Just to monitor issues as usual...

### Update checklist
